### PR TITLE
Initial layout

### PR DESCRIPTION
**Pull request by reshma-qburst**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -1,12 +1,107 @@
+* {outline: none;}
 body {font-family: Verdana, sans-serif; font-size: 0.8em;}
```
Comment (rekharv) [inline assets/css/styles.css]: Follow same alignment format throughout the project:-
either in a single line (like header,section,footer {border: 1px solid grey; margin: 5px; padding: 8px;}) or 
each styles in separate lines (like header,footer {
        color: #fff;
  	background-color: #444;
  	margin-bottom: 5px;
  })
Mostly we follow second approach

File: assets/css/styles.css [hunk 2]
```diff
@@ -1,12 +1,107 @@
...
+  
+ 	-webkit-filter: blur(5px);
+ 	-moz-filter: blur(5px);
+ 	-o-filter: blur(5px);
+ 	-ms-filter: blur(5px);
+ 	filter: blur(5px);
+}
+#index,#index .tabcontents,#gallery,#success,#success .tabcontents {background-color: #ffffb2;}
+#content1 .figure {margin: 0; text-align:center; text-align: -moz-center;}
+article,footer,.tabcontents h3 {text-align:center; text-align: -moz-center;}
```
Comment (rekharv) [inline assets/css/styles.css]: Always put a space after comma

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
```
Comment (rekharv) [inline contact.html]: Indent the <head> and <body> tags with a tab space as it is inside the <html> tag

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	<main>
+		<section id="content3">
+				<h2>Registration Form</h2>
+				<p>
+					Below is the <mark>form</mark> for <bdi><i>registration</i></bdi>
+				</p>
+				<div>
+					<form method="post" action="success.html" class="contactForm">
+						<div>
+							<label>First Name:</label>
```
Comment (rekharv) [inline contact.html]: Make use of the "for" attribute for the label tags

File: assets/css/styles.css (hunk 1 above)
Comment (reshma-qburst) [inline assets/css/styles.css]: Fixed

File: contact.html (hunk 4 above)
Comment (reshma-qburst) [inline contact.html]: Fixed

File: contact.html (hunk 3 above)
Comment (reshma-qburst) [inline contact.html]: Fixed

File: assets/css/styles.css (hunk 2 above)
Comment (reshma-qburst) [inline assets/css/styles.css]: Fixed

File: assets/css/styles.css (hunk 1 above)
Comment (reshma-qburst) [inline assets/css/styles.css]: Fixed

**Pull request merged**